docs/intro.md

```diff
@@ -73,16 +73,16 @@ mounted in a `snappy` capable system.
 
 ### Prime
 
-The prime step moves the data into a `./snap` directory. It contains only
+The prime step moves the data into a `snap/prime` directory. It contains only
 the content that will be put into the final snap package, unlike the staging
 area which may include some development files not destined for your package.
 
 The Snappy metadata information about your project will also now be placed in
-`./snap/meta`. Snapcraft takes care of generating all the meta-data Snappy
+`snap/prime/meta`. Snapcraft takes care of generating all the meta-data Snappy
 expects. For a breakdown of what this is, have a look at our [Snappy developer
 reference](http://snapcraft.io/docs/build-snaps/metadata).
 
-This `./snap` directory is useful for inspecting what is going into your snap
+This `snap/prime` directory is useful for inspecting what is going into your snap
```

Comment:
I can't comment on it since it's not part of the diff, but you probably want to update [line 93](https://github.com/snapcore/snapcraft/pull/1073/files#diff-18d58d2666f9af2b2086e0baa55f315dR93), below, as well.